improver/utilities/ancillary_creation.py

```diff
@@ -258,6 +261,68 @@ def __repr__(self):
                                             self.t_increment))
         return result
 
+    def saturation_vapour_pressure_goff_gratch(self, temperature):
+        """
+        Saturation Vapour pressure in a water vapour system calculated using
+        the Goff-Gratch Equation (WMO standard method).
+
+        Args:
+            temperature (numpy.ndarray):
+                Temperature values in Kelvin. Valid from 173K to 373K
+
+        Returns:
+            numpy.ndarray:
+                Corresponding values of saturation vapour pressure for a pure
+                water vapour system, in hPa.
+
+        References:
+            Numerical data and functional relationships in science and
+            technology. New series. Group V. Volume 4. Meteorology.
+            Subvolume b. Physical and chemical properties of the air, P35.
+        """
```

Comment:
Moved from `psychrometric_calculations.Utilities`, and modified minimally to accept a temperature array rather than a cube.